runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/PipelineResourcesTest.java

```diff
@@ -23,56 +23,54 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.net.URL;
-import java.net.URLClassLoader;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import org.apache.beam.vendor.guava.v20_0.com.google.common.collect.ImmutableList;
+import org.junit.AfterClass;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import org.mockito.Mockito;
 
 /** Tests for PipelineResources. */
 @RunWith(JUnit4.class)
 public class PipelineResourcesTest {
 
+  private static final String classPath = System.getProperty("java.class.path");
```

Comment:
Add test rule [RestoreSystemProperties](https://github.com/apache/beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/testing/RestoreSystemProperties.java)

Reply:
It's part of the test scope in java-core. Maybe move the rule to `test-utils`?

Reply:
Sounds like a good idea.

It would be good if we could move as many of the files in https://github.com/apache/beam/tree/master/sdks/java/core/src/test/java/org/apache/beam/sdk/testing over to it.